### PR TITLE
check that there is an existing location

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?182632

A daily emailed report is erroring when `self.get_locations` is empty.

@orangejenny @kkrampa @esoergel

### CODE REVIEW

**Comment by orangejenny** [comment]
looks fine pending tests

File: custom/ewsghana/reports/specific_reports/reporting_rates.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def headers(self):
     @property
     def rows(self):
         rows = []
-        if self.location_id:
+        if self.location_id and self.get_locations:
```
Comment (czue) [inline custom/ewsghana/reports/specific_reports/reporting_rates.py]: I know you didn't write it, but it's pretty confusing to have a property start with `get_`. boy scout rule and what not

Comment (nickpell) [inline custom/ewsghana/reports/specific_reports/reporting_rates.py]: property renamed

**Comment by czue** [comment]
nice, thanks. g2m when tests pass

**Comment by nickpell** [comment]
:white_check_mark:

**Pull request merged**